### PR TITLE
Fix RPC curl commands

### PR DESCRIPTION
**Pull request by atomaka**
* Corrects protocol from http to https on all curl commands on the "Powered By" pages
* Fixes syntax errors in `xcmpHandler_crossChainAccount` examples on both "Powered by" pages
* Corrects method name for querying task ids (RPC only for auto compounding tasks)
* Replaces RPC under "Get a task by taskId" with documentation for `automationTime_generateTaskId`
* Formats curl commands in both "Powered By" pages for easier viewing

Formatting on other curl commands could be improved but I only focused on the commands that were not functioning here.

### CODE REVIEW

File: _docs/powered-by-oak-price-apis.md [hunk 1]
```diff
@@ -38,15 +38,20 @@ The following requirements apply for any Kusama parachain that wishes to support
...
+  --header 'Content-Type: application/json' \
+  --data-raw '{
+   "id":1,
+   "jsonrpc":"2.0",
+   "method": "xcmpHandler_crossChainAccount",
+   "params": ["SS58_ACCOUNT_ID"]
+ }'
 ```
 
+Replace `SS58_ACCOUNT_ID` with the account you want to lookup.
```
Comment (irsal) [inline _docs/powered-by-oak-price-apis.md:53]: does it have to be in a specific prefix? e.g. `51`?

**Review by irsal — APPROVED** [review]
one q

File: _docs/powered-by-oak-price-apis.md (hunk 1 above)
Comment (atomaka) [reply _docs/powered-by-oak-price-apis.md:53]: This one does not.  The RPC will take in a wallet with any SS58 prefix and return the correct address with a substrate network prefix (`42`):

```
Querying with OAK network '6871Utgdq5ycWnTzm7VEwQfg4zM81JqYv8EDzKPu3fdzsXK8'
{"jsonrpc":"2.0","result":"5Egie3jrkENmFv5tUUcFCp72zz6xeL8JzJ8hjvKuuGcQySGD","id":1}
Querying with Substrate network '5E259xp1NU757FppcKa2YJGrFd4UWfAsDfF3TAcBsyHcLryL'
{"jsonrpc":"2.0","result":"5Egie3jrkENmFv5tUUcFCp72zz6xeL8JzJ8hjvKuuGcQySGD","id":1}
```

As far as I've tested, SS58 prefixed addresses can be used interchangeably cross chain with both the Polkadot Wallet App and the Polkadot.js API.

**Review by nvengal — APPROVED** [review]

**Comment by irsal** [comment]
merge conf

**Comment by atomaka** [comment]
@irsal https://github.com/OAK-Foundation/OAK-docs/pull/90 deleted several of the curl commands. I'm not sure if this was intentional or not. Resolution is unclear.

**Comment by atomaka** [comment]
I've formatted the RPC calls that are present in the newest version. This differences between this branch and current main were:

* Example for `automationTime_getTimeAutomationFees` was added
* Examples for `automationTime_generateTaskId` and `xcmpHandler_crossChainAccount` were still present
* Examples for `xcmpHandler_fees` and `automationTime_getAutoCompoundDelegatedStakeTaskIds` were removed

**Review by irsal — APPROVED** [review]
lgtm

**Pull request merged**